templates/Dataland/datasetlinks.html.twig

```diff
@@ -0,0 +1,14 @@
+<h2>Dataset Links:</h2>
+<div>
+    {% for link in dataset.datasetSubmission.datasetLinks %}
+        <p>
+            <b>Name:</b>{{ link.name }}<br>
+            <b>Description:</b>{{ link.description }}<br>
+            <b>Function:</b>{{ link.functionCode }}<br>

```

Comment:
we just showing the raw function code here, or turning it into it's textual representation?

Reply:
Was working on that, now fixed.